lib/http2.c

```diff
@@ -1203,6 +1203,13 @@ void Curl_http2_done(struct Curl_easy *data, bool premature)
     }
     http->stream_id = 0;
   }
+
+  if (0 == nghttp2_session_check_request_allowed(httpc->h2)) {
+    // No more requests are allowedi, indicating that this connection
+    // may no be reused. This is set when GOAWAY has been received or
+    // when no more stream identifiers can be allocated.
```

Comment:
You just need do change the comment to `/* style */` to be C90 compliant, otherwise I approve!

Reply:
Done.